doc/index.rst

```diff
@@ -101,6 +101,6 @@
    user_guide.rst
    auto_examples/index.rst
    whats_new.rst
-   contributing.rst
```

Comment:
If we're no longer linking CONTRIBUTING.rst in the docs should we convert to md so it appears cleanly in the github UI ?

Reply:
We are including it, via the "include" directive, at the end of development.rst

Reply:
Right, sorry, I just remember there were concerns about any of the guides getting too long. I was wondering if it would make sense to link out rather than include it ? No strong opinion, just an idea !